libs/dh/market-participant/feature-market-participant/src/components/details/details.component.html

```diff
@@ -14,54 +14,63 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-@let selectedActor = actor();
-
-<ng-container *transloco="let t; read: 'marketParticipant.actorsOverview.drawer'">
-  <watt-drawer [loading]="isLoading()" #drawer (closed)="closed.emit()" [size]="drawerSize()">
+@let loadedMarketParticipant = marketParticipant();
+
+<ng-container *transloco="let t; prefix: 'marketParticipant.actorsOverview.drawer'">
+  <watt-drawer
+    autoOpen="true"
```

Comment:
nitpick: shorter syntax. Works as well.


```suggestion
    autoOpen
```